README.md

```diff
@@ -4,9 +4,11 @@ Demo to show a real-time trading dashboard for three different exchanges.
 
 The complete crypto-trading demo has two components:
 
-1. A stream application.
-2. A UI application written in Reactjs
+1. A backend built with Macrometa a Document Collection, multiple Streams, and a Stream Worker.
 
+If you haven't already configure the back end. Here's a link to the [Quick Start Guide](https://macrometa.com/docs/apps/crypto-trading).
+
+2. A headless frontend written in Reactjs.
 
 The UI then makes use of all the streams and the `trades` collection to show the charts and suggestions at one place.
```

Comment:
You removed mention of the UI, so I am wondering what UI. Also, no context for the `trades` collection at this point in the document.